stylesheets/_component.navigation.scss

```diff
@@ -350,18 +356,45 @@ $nav-color-dark:  darken(color(jadu-blue, dark), 5%) !default;
         &:hover,
         &:focus {
             background-color: darken($nav-color-dark, 4%);
-            color: white;
+            color: color(white);
         }
 
         &:active {
-            background-color: $white;
+            background-color: color(white);
             color: $nav-color-dark;
         }
     }
 }
 
+.nav-tertiary {
+    @extend .nav-secondary;
+
+    .nav-link {
+
+        span {
+            margin-left: 10px;
+        }
+    }
+}
+
+.nav-quaternary {
+    @extend .nav-secondary;
```

Comment:
Here too, can we abstract out into common styles / multiple sector rather than using extend which will bloat the css.